### ISSUE TITLE
Update NPM-version

### ISSUE THREAD

**Issue by duichwer** [aratcliffe/Leaflet.contextmenu#51]
I included leafelt-contextmenu in my project and after a long search i found out, that the new changes/commits arent't yet at npm available. I know i can donload it form github and build i myself, but wouldn't it be useful to update ?
I wanted to use relatedTarget.

**Comment by aratcliffe** [comment]
Hi, what's the procedure to upgrade the NPM version? Let me know and I'll make them.

**Comment by duichwer** [comment]
At first you have to update your version number in your package.json. 
For choosing the right version numer you should think about what you have changed since the last version and which influence this might have (e.g. are there only patches?) For more Info about semantic versioning you could have a look on [npmjs](https://docs.npmjs.com/getting-started/semantic-versioning)
I might be useful to tag that commit with that version number, in case to find out easily which version belongs to which commit . 

After you have done that navigate into your Repository.
Now you can publish your new version with:
`npm publish`

This should work for you. There are optional parameters, that I think you don't need. For more info have a look into  the [npm-doc](https://docs.npmjs.com/cli/publish).

To summarize:

1. Change version number in package.json
2. Tag our commit with that number
3. Run `npm publish` in your Repository

I apologize for my english.

**Comment by duichwer** [comment]
I think, I forgot, that before publishing you should push a current version of the dist-folder.

**Comment by aratcliffe** [comment]
Sorry for the delay, I've done this now. Let me know if there are any problems.
